src/elements/ScheduleRecurring/ScheduleRecurring.js

```diff
@@ -29,63 +26,56 @@ const ScheduleRecurring = ({
     }
   } = recurringState
 
-  const handleRangeInputChange = item => {
-    switch (scheduleRepeatActiveOption) {
+  const handleScheduleRepeatChange = (value, activeOption) => {
+    const selectedOption = activeOption || scheduleRepeatActiveOption
```

Comment:
If you just want to default `activeOption` to `scheduleRepeatActiveOption` in case `activeOption` is nullish (meaning either the `null` value or the `undefined` value) then you better use the `??` operator:
```suggestion
    const selectedOption = activeOption ?? scheduleRepeatActiveOption
```
This prevents non-obvious bugs where `0` `-0` `false` `NaN` and `''` might be valid values, but `||` operator will give its right operand and not the left:

```js
const foo = bar => {
  const flag = foo || true // default to true if bar is undefined
  console.log(`You sent: ${flag}`)
}
foo()
// => will correctly print 'you sent: true', because no parameter was sent, and it defaults to true
foo(true)
// => will correctly print 'you sent: true' because that was the provided parameter

foo(false)
// => will INCORRECTLY print 'you sent: true', instead of 'you sent: false'
```
If we use `??`:
```js
const foo = bar => {
  const flag = foo ?? true // default to true if bar is undefined
  console.log(`You sent: ${flag}`)
}
foo()
// => will correctly print 'you sent: true', because no parameter was sent, and it defaults to true
foo(true)
// => will correctly print 'you sent: true' because that was the provided parameter
foo(false)
// => will CORRECTLY print 'you sent: false', because that was the provided parameter
```